app/policies/project_policy.rb

```diff
@@ -28,7 +28,7 @@ def resolve
       elsif is_admin?
         current_team.projects
       else
-        current_user.projects.not_archived.where(id: current_team.projects.pluck(:id))
+        current_user.projects.not_archived.where(id: current_team&.projects&.pluck(:id))
```

Comment:
But we will get an error: `undefined method not_archived for []:Array`, because in controller we have: `@projects = policy_scope(Project).not_archived`